services/src/main/java/org/keycloak/protocol/oid4vc/model/VerifiableCredential.java

```diff
@@ -0,0 +1,96 @@
+package org.keycloak.protocol.oid4vc.model;
```

Comment:
Could you add header comments the same as other class files if you add a new class?

Reply:
I am sorry not to tell you what the header comments means exactly.

I intended to add the following header comments.

```
/*
 * Copyright 2024 Red Hat, Inc. and/or its affiliates
 * and other contributors as indicated by the @author tags.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

Reply:
No problem, I added them [8200521](https://github.com/keycloak/keycloak/pull/26263/commits/82005216d874fa5e01a2d2e63221f056536682ae)

Reply:
I am afraid the added comments include:
```
* Copyright 2016 Red Hat, Inc. and/or its affiliates
```
How about the following?
```
* Copyright 2024 Red Hat, Inc. and/or its affiliates
```

Reply:
Fixed [a36c099](https://github.com/keycloak/keycloak/pull/26263/commits/a36c099eb305125846d25f1d73493dc56556573a)

Reply:
Thank you. I confirmed the revised codes.